src/main/java/net/dries007/tfc/objects/blocks/wood/BlockSupport.java

```diff
@@ -0,0 +1,392 @@
+    {
+        return new BlockStateContainer(this, AXIS, NORTH, SOUTH, EAST, WEST);
+    }
+
+    /**
+     * Checks if this support block can support collapsable/fallable blocks
+     * Returns true only if this is horizontally placed and can stay in place.
+     *
+     * @param world the worldObj this support block is in
+     * @param pos   the BlockPos this support block is in
+     * @return true if this can support blocks
+     */
+    public boolean canSupportBlocks(IBlockAccess world, BlockPos pos)
+    {
+        return canBlockStay(world, pos) && getAxis(world, pos) != BlockLog.EnumAxis.Y;
+    }
+
+    /**
+     * Check if the facing can connect (ie: is another support)
+     *
+     * @param world  the worldObj to check
+     * @param pos    the BlockPos the current block is in
+     * @param facing the facing to check for connection
+     * @return true if the facing has another support block
+     */
+    private boolean isConnectable(IBlockAccess world, BlockPos pos, EnumFacing facing)
+    {
+        BlockPos other = pos.offset(facing);
+        return world.getBlockState(other).getBlock() instanceof BlockSupport; //Instanceof so it can connect different wood types
+    }
+
+    /**
+     * Checks if this BlockPos can only be vertical
+     *
+     * @param world worldObj to check
+     * @param pos   BlockPos to check
+     * @return true if this block can only be vertical, false otherwise
+     */
+    private boolean isVertical(IBlockAccess world, BlockPos pos)
+    {
+        //If above or below is support, this one must be vertical too
+        if (isConnectable(world, pos, EnumFacing.UP) || isConnectable(world, pos, EnumFacing.DOWN))
+            return true;
+        //If this is an intersection, this one must be vertical
+        return (isConnectable(world, pos, EnumFacing.NORTH) || isConnectable(world, pos, EnumFacing.SOUTH))
+            && (isConnectable(world, pos, EnumFacing.EAST) || isConnectable(world, pos, EnumFacing.WEST));
+    }
+
+    /**
+     * Return the axis for this block
+     *
+     * @param world the worldObj this block is in
+     * @param pos   this block's position
+     * @return axis value
+     */
+    private BlockLog.EnumAxis getAxis(IBlockAccess world, BlockPos pos)
+    {
+        if (isVertical(world, pos)) return BlockLog.EnumAxis.Y;
+        if (isConnectable(world, pos, EnumFacing.NORTH) || isConnectable(world, pos, EnumFacing.SOUTH))
+            return BlockLog.EnumAxis.Z;
+        if (isConnectable(world, pos, EnumFacing.WEST) || isConnectable(world, pos, EnumFacing.EAST))
+            return BlockLog.EnumAxis.X;
+        //If none of the above, this is a new placement without any neighboring support blocks
+        return BlockLog.EnumAxis.Y;
+    }
+
+    private boolean canBlockStay(IBlockAccess world, BlockPos pos)
+    {
+        BlockLog.EnumAxis axis = getAxis(world, pos);
+        if (axis == BlockLog.EnumAxis.Y)
+        {
+            return !world.isAirBlock(pos.down());
```

Comment:
This should be more than just not air - supports require a solid block below them